lib/array_equals.rb

```diff
@@ -1,5 +1,29 @@
 # Determines if the two input arrays have the same count of elements
 # and the same integer values in the same exact order
 def array_equals(array1, array2)
+true_false_array = []
```

Comment:
It seems a little wasteful to build an entire array here.  Not so efficient. It does work however.